content/en/data-playground/dc-metro.md

```diff
@@ -33,7 +33,7 @@ weight: 5
 
 {{% data-playground-code-tab tabIndex="go" name="Go"  %}}
 
-```go {linenos=table,hl_lines=[1,"3-4"]}
```

Comment:
This was removed in a PR last week. Could we agree to make sure a branch is updated with changes from main when creating PRs? I understand that changes can be merged while a PR is waiting for review and that we can create PRs around the same time. However, I'm thinking more about changes that occur a day or more before a PR is created.